resources/js/components/ui/Table/tquery_filters/DateTimeFilterControl.tsx

```diff
@@ -18,21 +18,35 @@ export interface DateTimeRangeFilter extends BoolOpFilter {
   val: RangeSideFilter[];
 }
 
-interface Props extends FilterControlProps<DateTimeRangeFilter> {
-  /** The data type, datetime by default. */
-  columnType?: "date" | "datetime";
+interface DateTimeColumnProps extends FilterControlProps<DateTimeRangeFilter> {
+  columnType?: "datetime";
+  /** Whether the inputs should only set date (and no time). Default is false. */
+  useDateOnlyInputs?: boolean;
 }
 
+interface DateColumnProps extends FilterControlProps<DateTimeRangeFilter> {
+  columnType: "date";
+}
+
+type Props = DateColumnProps | DateTimeColumnProps;
+
 export const DateTimeFilterControl: Component<Props> = (props) => {
   const t = useLangFunc();
-  const inputType = () => (props.columnType === "date" ? "date" : "datetime-local");
+  const columnType = () => props.columnType || "datetime";
+  const inputsType = () => (props.columnType === "date" || props.useDateOnlyInputs ? "date" : "datetime-local");
   function toInputValue(date: Date | undefined) {
-    return date ? (props.columnType === "date" ? toLocalDateISOString(date) : toLocalISOString(date).slice(0, -3)) : "";
+    return date
+      ? inputsType() === "date"
+        ? toLocalDateISOString(date)
+        : toLocalISOString(date)
+            // Cut off seconds.
+            .slice(0, -3)
+      : "";
   }
-  const findVal = (op: RangeOp) => {
```

Comment:
czemu nagła zmiana z arrow function na zwykłe? Nie mówię, że to złe, tylko ciekawi mnie zmiana podejścia